docs/rest.md

```diff
@@ -661,3 +661,25 @@ Sample input:
     “slot”: {“snap”: “keyboard-lights”, “slot”: “capslock-led”}
 }
 ```
+
+## /2.0/events
+
+### GET
+
+* Description: Websocket upgrade
```

Comment:
That was lifted from the [Snappy over REST](https://docs.google.com/document/d/1VqrEK3V_7jYMICy09tOVSaATb5KWu6Vs3CbDu9Nh1cQ/edit#heading=h.d10cqykq8uh0) doc. Would the following be better or is it too much?

"upgrade the connection to a websocket on which notifications will be sent"